platform/darwin/src/MGLLight.h

```diff
@@ -0,0 +1,125 @@
+#import <CoreGraphics/CoreGraphics.h>
+
+#import "MGLFoundation.h"
+#import "MGLStyleValue.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+
+/** Options to specify extruded geometries are lit relative to the map or viewport. */
+typedef NS_ENUM(NSUInteger, MGLLightAnchor) {
+    /** The position of the light source is aligned to the rotation of the map. */
+    MGLLightAnchorMap,
+    /** The position of the light source is aligned to the rotation of the viewport. */
+    MGLLightAnchorViewport
+};
+
+/**
+ A structure containing information about the position of the light source
+ relative to lit geometries.
+ */
+typedef struct MGLSphericalPosition {
+    /** Distance from the center of the base of an object to its light. */
+    CGFloat radial;
```

Comment:
What unit is this distance in? If this distance is measured in meters, its type should be CLLocationDistance.